.github/classroom.rb

```diff
@@ -0,0 +1,15 @@
+class ClassRoom
+    attr_accessor :label
+    attr_reader :students
+  
+    def initialize(label)
+      @label = label
+      @students = []
+    end
+  
+    def add_students(student)
+      @students.push(student)
+      student.classroom = self
+    end
+  end
```

Comment:
- [ ] This class should not be in the `.github` directory please make it in the root of your project like other classes